Dockerfile

```diff
@@ -37,8 +37,13 @@ RUN bitrise stepman update
 # releases: https://github.com/npm/cli/releases
 RUN npm install -g npm@6.13.4
 
-# --- Pre-install jq
-RUN sudo apt-get -y install jq
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
+    jq \
+    awscli
+
```

Comment:
why so much white space?

Reply:
It's a good practice to break it into multiple lines, so in the future we can add more tools here.

Reply:
I know, what I meant is that there was like 5 new lines :)